logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/log/LogStreamBatchWriterImpl.java

```diff
@@ -51,6 +52,7 @@ final class LogStreamBatchWriterImpl implements LogStreamBatchWriter, LogEntryBu
 
   private final Dispatcher logWriteBuffer;
   private final int logId;
+  private boolean skipProcessing;
```

Comment:
:thinking: It is a bit weird, this is actually just for one entry right? I guess this is how we did that previously...not optimal :sweat_smile:

Reply:
Yeah it is :+1: